liblauncher/reg.cpp

```diff
@@ -41,4 +41,30 @@ extern "C" JNIEXPORT jstring JNICALL Java_net_runelite_launcher_Launcher_regQuer
     }
 
     return env->NewString(reinterpret_cast<const jchar *>(pvData), pcbData / 2 - 1);
+}
+
+extern "C" JNIEXPORT void JNICALL Java_net_runelite_launcher_FilePermissionManager_regDeleteValue(JNIEnv *env, jclass clazz, jstring keyObj, jstring subKeyObj, jstring valueObj) {

```

Comment:
you could consider making this return jboolean so we can tell if it is successful